can-stache.js

```diff
@@ -99,8 +99,9 @@ function stache (filename, template) {
 					section.endSection();
 				}
 
-				if(section instanceof HTMLSectionBuilder) {
-					//!steal-remove-start
+				// to avoid "Blocks are nested too deeply" when linting
+				//!steal-remove-start
+				if(section instanceof HTMLSectionBuilder && process.env.NODE_ENV !== 'production') {
```

Comment:
The `process.env.NODE_ENV !== 'production'` needs to wrap around this if block, otherwise it won't be removed during the build.  It should be:

```js
if(process.env.NODE_ENV !== 'production') {
  if(section instanceof HTMLSectionBuilder) {
    // .....
  }
}
```

Same way any time you add these conditions.

Reply:
Thanks, I fixed it!

Reply:
But jshint complains:
```can-stache.js: line 108, col 29, Blocks are nested too deeply. (5)```
That's why I want to put it in one line